src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1468,6 +1468,10 @@ trait Definitions extends api.StandardDefinitions {
       lazy val StringContext_raw = getMemberMethod(StringContextClass, nme.raw_)
       lazy val StringContext_apply = getMemberMethod(StringContextModule, nme.apply)
 
+      lazy val Predef_=:= = getMemberClass(PredefModule, TypeName("=:="))
+      lazy val Predef_<:< = getMemberClass(PredefModule, TypeName("<:<"))
+      lazy val Predef_Dummy = getMemberClass(PredefModule, TypeName("DummyImplicit"))
```

Comment:
Should these be standard `tpnme`s? Would it be worth it? Who knows?

Reply:
Also,

```
error: object Predef does not have a member =:=
```

Reply:
I have no sense about these things. I learned the hard way that you can use string interpolation in Definitions because of the definition of `StringContext` there.

Reply:
Dunno why, but it's got a member class `$eq$colon$eq`.

```scala
scala> getMemberClass(PredefModule, TypeName("=:=").encodedName)
res6: $r.intp.global.ClassSymbol = class =:=
```

Reply:
Yes, encoding crossed my mind.

OK, I just checked and `???` in StdNames does encode first.

I was too lazy because to grep for it I have to escape it...

Reply:
...also I need more sleep.